### DISCUSSION TITLE
any roadmap?

### DISCUSSION THREAD

**Discussion by gormonn** [ReactUnity/core#58] [General]
Hi!
It's wonderful!
Do you have some roadmap?

**Comment by gkurt** [comment]
Hello. There isn't a clear roadmap. I am working on performance, testing and documentation improvements before releasing v1. Other than that, the project is fully functional and working as intended. What kind of roadmap would you like to see?

**Reply by gormonn** [reply]
If so, then that's cool. Good luck!

**Reply by gkurt** [reply]
Most of the discussions generally happen in the Discord server. You can also check that out.
https://discord.gg/czYqBzYu

**Reply by gormonn** [reply]
Thanks for the invite.
In my picture of the world, the presence of discussions inside the repo gives more reason to believe that the project is alive.
